spec/requests/wikies_spec.rb

```diff
@@ -1,10 +1,18 @@
 require 'rails_helper'
-
+include Devise::TestHelpers
```

Comment:
This `include` can be removed by configuring RSpec to include `Devise::TestHelpers` in all request specs.

```
# Replaces similar line in devise_support.rb:
config.include Devise::TestHelpers, [type: :controller, type :request]
```